test/integration/scripts/test_clean.py

```diff
@@ -0,0 +1,232 @@
+    """
+    exp = autosubmit_exp(_EXPID, experiment_data={})
+
+    plots_dir = Path(exp.as_conf.basic_config.LOCAL_ROOT_DIR, f'{_EXPID}/plot/')
+
+    for i, plot in enumerate([
+        'plot_a.pdf', 'plot_b.pdf', 'plot_c.pdf', 'plot_d.pdf', f'{_EXPID}_statistics_1.pdf',
+        f'{_EXPID}_statistics_2.pdf', f'{_EXPID}_statistics_3.pdf'
+    ]):
+        p = Path(plots_dir / plot)
+        p.touch()
+        utime(str(p), (i, i))
+
+    mocker.patch('sys.argv', ['autosubmit', 'clean', _EXPID, '--stats'])
+    main()
+
+    plots = list(plots_dir.iterdir())
+
+    # keeps plot files, and two newest statistics files
+    assert len(plots) == 6
+    assert Path(plots_dir / f'{_EXPID}_statistics_2.pdf').exists()
+    assert Path(plots_dir / f'{_EXPID}_statistics_3.pdf').exists()
+
+
+def test_clean_dummy_project(autosubmit_exp, mocker):
+    """Test cleaning an Autosubmit experiment dummy project.
+
+    Autosubmit only deletes Git projects, so it must simply log and exit,
+    without touching plot or statistics files.
+    """
+    mocked_log = mocker.patch('autosubmit.autosubmit.Log')
+
+    exp = autosubmit_exp(_EXPID, experiment_data={
+        'PROJECT': {
+            'PROJECT_TYPE': 'none'
+        }
+    })
+
+    Path(exp.as_conf.basic_config.LOCAL_ROOT_DIR, _EXPID, ).mkdir(exist_ok=True)
+
+    plots_dir = Path(exp.as_conf.basic_config.LOCAL_ROOT_DIR, f'{_EXPID}/plot/')
+
+    for i, plot in enumerate([
+        'plot_a.pdf', 'plot_b.pdf', 'plot_c.pdf', 'plot_d.pdf', f'{_EXPID}_statistics_1.pdf',
+        f'{_EXPID}_statistics_2.pdf', f'{_EXPID}_statistics_3.pdf'
+    ]):
+        p = Path(plots_dir / plot)
+        p.touch()
+        utime(str(p), (i, i))
+
+    mocker.patch('sys.argv', ['autosubmit', 'clean', _EXPID, '--project'])
+    main()
+
+    plots = list(plots_dir.iterdir())
+
+    # keeps plots and statistics files
+    assert len(plots) == 7
+
+    assert mocked_log.info.called
+    assert 'No project to clean' in mocked_log.info.call_args_list[-1].args[0]
+
+
+@pytest.mark.parametrize(
+    'clean_git_return,main_exit_code,expected_result,expected_info',
+    [
+        (True, 0, 'Git project cleaned', None),
+        (False, 1, None, 'Cleaning GIT directory'),
+    ],
+    ids=[
+        'Best scenario',
+        'clean_git fails, so no result logged'
+    ]
+)
+def test_clean_git_project(
+        clean_git_return: bool,
+        main_exit_code: int,
+        expected_result: str,
+        expected_info: str,
+        autosubmit_exp,
+        mocker,
+        tmp_path):
+    """Test cleaning an Autosubmit experiment Git project.
+
+    Autosubmit only deletes Git projects, so it must simply log and exit,
+    without touching plot or statistics files.
+    """
+    mocked_log = mocker.patch('autosubmit.autosubmit.Log')
+
+    # TODO: Bug in AutosubmitConfigParser, can be deleted once it's fixed,
+    #       https://github.com/BSC-ES/autosubmit-config-parser/issues/87.
```

Comment:
Another :bug: , but this time I won't fix this as it's in config-parser and I have to go back to the PostgreSQL branch :+1: